gradle/prCheck.gradle

```diff
@@ -0,0 +1,95 @@
+//
+// Validate Pull Request metadata
+//
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.github.ricksbrown:cowsay:1.1.0'
+    }
+}
+
+import groovy.json.JsonSlurper
+import com.github.ricksbrown.cowsay.Cowsay
+
+def GOOD_PR_PROCESS = """
+How to pass this check?
+    1. Always have a Jira before you start working on a task (otherwise you cannot know what branch to start from).
+    2. Make sure the Jira has a 'Fix Version' defined. If it doesn't consult the team what is a good fix version for your change.
+    3. If you are working on a Jira with 'Fix Version' ending with '.1' you are working on a change that should go to a master branch. 
+       All other changes should go to the (current) fix release branch. Such branches are prefixed 'rel-'. 
+    4. Put the Jira to 'In Progress' state.
+    5. Make a code change. Include Jira ticket ID in message of the main commit (this is not checked). 
```

Comment:
What is main commit? I would write ... of the all commits ...

Reply:
Jira then makes a links for each of the commits

Reply:
I changed the wording to "one of your commits" to avoid confusing term "main commit"